Formula/lftp.rb

```diff
@@ -1,23 +1,33 @@
 class Lftp < Formula
   desc "Sophisticated file transfer program"
   homepage "https://lftp.yar.ru/"
-  url "https://lftp.yar.ru/ftp/lftp-4.6.5.tar.xz"
-  sha256 "1fd0920a1791ce0e9e39ffce77ae6619e5dc665f16e9380bafbfc69411eeb71e"
+  url "https://lftp.yar.ru/ftp/lftp-4.7.2.tar.xz"
+  sha256 "475d7699b1390f951efee867ba1ad600f78329e13fd2a04d92f82bfffb70d872"
 
   bottle do
     sha256 "f341d62601963f968e46907f88e1e96d21154deb2b60d97bd3bac27a37816575" => :el_capitan
     sha256 "182979ca79b7ea9d9e70e9fefab990d5d238da709ce0e17edf5ec7166b976fb1" => :yosemite
     sha256 "dc44f63a8f1767af2b4ed2008c199377eabd117339f01ee17bb73aa844cf3dee" => :mavericks
   end
 
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
   depends_on "pkg-config" => :build
   depends_on "readline"
-  depends_on "openssl"
+  depends_on "openssl" if MacOS.version < :yosemite
```

Comment:
I ran it on El Cap and it's still trying to use `openssl`, which is now an error when it tries to use the system `openssl`. That means this dependency should not be limited to `< :yosemite`.

```

In file included from FileAccess.cc:1021:
In file included from ./ftpclass.h:29:
./lftp_ssl.h:27:12: fatal error: 'openssl/ssl.h' file not found
#  include <openssl/ssl.h>
           ^
/bin/sh ../libtool --silent  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I../lib  -I../lib -I../trio   -O2 -Wall -Wwrite-strings -Woverloaded-virtual -fno-exceptions -fno-rtti -fno-implement-inlines -c -o StringSet.lo StringSet.cc
1 error generated.
make[2]: *** [FileAccess.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [install-recursive] Error 1
make: *** [install] Error 2
```